File: Week 1 - C/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {

    int height = get_int("Height? ");

    while(height < 1 || height > 8) {
        height = get_int("Height? ");
    }
    

    for(int i = 0; i < height; i++){
        
        for(int k = height - 1; k > i; k--){
            printf(" ");
        }
            
        for(int j = 0; j <= i; j++){
            printf("#");
        }
        
        printf("  ");
        
        for(int l = 1; l <= i; l++){
            printf("#");
        }
        
        printf("#\n");
        
    }
}

File: Week  6 - Python/sentimental-mario/mario.py
from cs50 import get_int

height = get_int("Height?\n")

while height < 1 or height > 8:
    height = get_int("Height?\n")

for i in range(height):
    for k in range(height - 1, i, -1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for n in range(i + 1):
        print("#", end="")
    print("")
